.github/workflows/build_windows.yml

```diff
@@ -0,0 +1,185 @@
+name: Build Windows
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '**.md'
+    tags:
+      - '*'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build-windows:
+    runs-on: windows-latest
+    # https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup devcmd
+        uses: ilammy/msvc-dev-cmd@v1
+
+      # https://learn.microsoft.com/en-us/vcpkg/users/binarycaching#gha
+      - name: Set variables for vcpkg
```

Comment:
What's the point of this? This step takes values from environment, and puts them to... environment?

Reply:
It sounds obscure to me, too.
I found this way to enable caching for vcpkg at the link https://learn.microsoft.com/en-us/vcpkg/users/binarycaching#gha , and it works.